consensus/spos/bls/subroundBlock.go

```diff
@@ -542,7 +532,7 @@ func (sr *subroundBlock) saveLeaderSignature(nodeKey []byte, signature []byte) b
 		log.Debug("saveLeaderSignature.ConsensusGroupIndex",
```

Comment:
now we are double printing the error messages. Here on all ifs and in the `printCancelRoundLogMessage` method

Reply:
fixed